src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -128,7 +128,9 @@ Assembly: 'SonarLint.VisualStudio.CFamily, Version=6.5.0.0, Culture=neutral, Pub
 Relative path: 'SonarLint.VisualStudio.CFamily.dll'
 
 Referenced assemblies:
+- 'Microsoft.VisualStudio.CoreUtility, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
 - 'Microsoft.VisualStudio.Shell.Framework, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
+- 'Microsoft.VisualStudio.Text.Data, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
```

Comment:
Types from these two VS interfaces are exposed by the `ISonarLangaugeRecognizer` interface so the `CFamily` assembly now depends on them too.